src/content/plugins/split-chunks-plugin.md

```diff
@@ -344,3 +345,28 @@ module.exports = {
 ```
 
 W> This might result in a large chunk containing all external packages. It is recommended to only include your core frameworks and utilities and dynamically load the rest of the dependencies.
+
+### Split Chunks: Example 3
+
+ Create a `custom vendors` chunk, which includes only those `node_modules` from the whole application which passes the RegEx Test
```

Comment:
I think this can be slightly reworded to something like: 
Create a `custom vendor` chunk, which contains certain `node_modules` packages matched by `RegExp`.